proxy/proxy.ts

```diff
@@ -0,0 +1,119 @@
+// @ts-ignore
+import isMartianIP from 'martian-cidr'
+import type http from 'node:http'
+import { createServer } from 'node:http'
+import { styleText } from 'node:util'
+
+/**
+ * Creates proxy server
+ *
+ * isProduction - main toggle for production features:
+ * - will allow only request that match productionDomainSuffix
+ *
+ * silentMode - will silent the output. Useful for testing
+ *
+ * hostnameWhitelist - Sometimes you need to allow request to certain ips like localhost for testing purposes.
+ *
+ * productionDomainSuffix - if isProduction, then only request from origins that match this param are allowed
+ *
+ * @example
+ * const p = createProxyServer()
+ * p.listen(5555).then(() => console.log('running on 5555'))
+ *
+ * @param { isProduction, silentMode, hostnameWhitelist, productionDomainSuffix } config
+ */
+const createProxyServer = (
```

Comment:
We do not use default export in the project (they sometimes have a problem in bundlers/tools, not a big dead but nice for code style—do we have ESLint rule for that?).

Let’s use `export function createProxyServer`

Reply:
Fixed, thanks!